src/sentry/incidents/endpoints/organization_alert_rule_index.py

```diff
@@ -129,9 +129,11 @@ def get(self, request: Request, organization) -> Response:
                 ),
             )
             issue_rules = issue_rules.annotate(date_triggered=far_past_date)
+        alert_rules_count = alert_rules.count()
+        issue_rules_count = issue_rules.count()
```

Comment:
@Neo-Zhixing is this going to give us the count of all alert rules for the org or just ones matching the filters?

Reply:
This is going to give the filtered counts

Reply:
Yeah and that'd be what you want - the count after the filter but before the pagination.

Reply:
@Neo-Zhixing right...makes sense to me.

Reply:
I don't think that's what you want. If the user filters alerts by name, then you'll get into a situation where you show the banner when there's an issue alert with the name, but no metric alert.

Reply:
Good point - I forgot that they could also filter by name. The other two criteria - project, and teams - make a lot of sense tho.

For now let's not complicate this further - I'll just do something on the frontend so that we don't show the banner when the user filter by name.